targets/zephyr/src/main-zephyr.c

```diff
@@ -135,9 +136,21 @@ static int shell_cmd_handler (int argc, char *argv[])
 
   if (jerry_value_has_error_flag (ret_val))
   {
-    printf ("Failed to run JS\n");
+    /* User-friendly error messages require at least "cp" JerryScript
+       profile. Include a message prefix in case "cp_minimal" profile
+       is used. */
+    printf ("Error executing statement: ");
+    /* Clear error flag, otherwise print call below won't produce any
+       output. */
+    jerry_value_clear_error_flag (&ret_val);
   }
 
+  jerry_value_t ret_val_print = jerry_call_function (print_function,
+    jerry_create_undefined (),
+    &ret_val,
+    1);
```

Comment:
@LaszloLango, The code above is taken as-is from main-unix.c. I'd be reluctant to stray too much from it, to avoid confusion for someone who may later deal with Zephyr code (including myself at some later time). I'd be happy to apply such change later, when main-unix.c will be updated. (Or can submit a next change to update both if you think it makes sense.)